addon-test-support/@ember/test-helpers/dom/scroll-to.ts

```diff
@@ -39,12 +38,6 @@ export default function scrollTo(
       throw new Error(`Element not found when calling \`scrollTo('${target}')\`.`);
     }
 
-    if (!isElement(element)) {
-      throw new Error(
```

Comment:
this is removed, cause `getElement(`, which stopped returning `Document` would fail if the element is not found. So we won't ever reach this  line.